addon/mixins/multi-list-controller.js

```diff
@@ -0,0 +1,291 @@
+import Ember from 'ember';
```

Comment:
You need to add the comment `@module` so that the autodocs is not written in vain.